api/scpca_portal/config/logging.py

```diff
@@ -50,3 +54,42 @@ def get_and_configure_logger(name: str) -> logging.Logger:
     logger.logger.addHandler(handler)
 
     return logger
+
+
+def get_formatted_time(timestamp: float) -> str:
+    """Format a timestamp to a local time."""
+    return datetime.fromtimestamp(timestamp).strftime("%H:%M:%S")
+
+
+def log_func_run_time(logger):
+    """
+    Log start time, end time, and duration of the wrapped function.
+    """
+
+    def decorator(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            global LOG_FUNC_RUN_TIMES
+            if not LOG_FUNC_RUN_TIMES:
+                return func(*args, **kwargs)
+
+            start_time = time.time()
+            logger.info(f"\nStart of function '{func.__module__}::{func.__name__}'.")
```

Comment:
```suggestion
            logger.info(f"\nEntering '{func.__module__}::{func.__name__}'.")
```

Reply:
You can create a variable for the module/function name and use it for entering and exiting logs.